ruby/vscode/minitest/tests.rb

```diff
@@ -35,9 +35,12 @@ def build_list
             full_path = File.expand_path(path, VSCode.project_root)
             path = full_path.gsub(VSCode.project_root.to_s, ".")
             path = "./#{path}" unless path.match?(/^\./)
+            description = test_name.gsub(/^test_/, "")
+            description = description.match?(/\s/) ? description : description.tr("_", " ")
```

Comment:
I think this'd be slightly faster in most cases since it wouldn't re-declare the variable when the match doesn't happen:

```suggestion
            description = description.tr("_", " ") unless description.match?(/\s/)
```

Although the Ruby interpreter might be smart enough to not redeclare it anyway 🤷‍♂️